Vagrantfile

```diff
@@ -63,6 +63,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       mount_options: synced_folder.include?('mount_options') ? synced_folder['mount_options'] : []
   end
 
+  # Provision using file provisioner to add known_hosts from host to guest.
+  config.vm.provision "file", source: "~/.ssh/known_hosts", destination: "~/.ssh/known_hosts"
```

Comment:
Will this work correctly on Windows? I haven't used the file provisioner with a file that may or may not exist, so haven't seen the behavior if the file doesn't exist.